libraries/GCS_MAVLink/GCS.h

```diff
@@ -418,6 +418,21 @@ class GCS
 
     void send_statustext(MAV_SEVERITY severity, uint8_t dest_bitmask, const char *text);
     void service_statustext(void);
+    virtual GCS_MAVLINK &chan(const uint8_t ofs) = 0;
+    virtual uint8_t num_gcs() const = 0;
+    void reset_cli_timeout();
+    void send_message(enum ap_message id);
+    void send_mission_item_reached_message(uint16_t mission_index);
+    void data_stream_send();
+    void update();
+    virtual bool cli_enabled() = 0;
+    virtual AP_HAL::BetterStream*  cliSerial() = 0;
```

Comment:
Is there any reason these two were made public in the base class and private in the subclasses? It's technically possible but not a good thing to do. I guess this is only used by the base class so it could be private in the base class.